packages/core/src/components/ChoiceList/ChoiceList.jsx

```diff
@@ -216,7 +217,11 @@ ChoiceList.propTypes = {
    * `checkbox` fields will be rendered. If less than 10 choices are passed in,
    * then `radio` buttons will be rendered.
    */
-  type: PropTypes.oneOf(['checkbox', 'radio', 'select'])
+  type: PropTypes.oneOf(['checkbox', 'radio', 'select']),
+  /**
+   * Adds `aria-label` attribute if component renders a select
```

Comment:
Given this is specific to `Select`, could you add this to `Select.propTypes` in `Select.jsx` as well?

Reply:
Good point. Fixed in a985582340118222c7f5df9b92c1ffc4d104f3b2.